account_invoice_supplier_ref_unique/models/account_invoice.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2016 Acsone
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
```

Comment:
Imports should be sorted Alphabetically
from odoo import api, fields, models, _